### PR TITLE
Add a random per session id token for load balancing multiple clients for single user.

### PR DESCRIPTION
**Pull request by Tilps**

### CODE REVIEW

File: lc0_main.go [hunk 1]
```diff
@@ -100,6 +102,7 @@ func getExtraParams() map[string]string {
 		"user":     *user,
 		"password": *password,
 		"version":  "17",
+		"id":       strconv.Itoa(randId),
```
Comment (borg323) [inline lc0_main.go]: I find "id" a bit misleading. Maybe "token" is better?

File: lc0_main.go [hunk 2]
```diff
@@ -697,6 +700,10 @@ func hideLc0argsFlag() {
 }
 
 func main() {
+	randBytes := make([]byte, 2)
+	rand.Reader.Read(randBytes)
+	// Ignoring errors from above is deliberate, and unsafe - care should be taken on server before relying on the id field.
```
Comment (borg323) [inline lc0_main.go]: Why not check for errors and set randId to -1?

File: lc0_main.go (hunk 1 above)
Comment (Tilps) [reply lc0_main.go]: Done.

File: lc0_main.go (hunk 2 above)
Comment (Tilps) [reply lc0_main.go]: Done - I hope it never errors anyway :)

**Pull request merged**